src/spi.rs

```diff
@@ -132,6 +96,9 @@ mod embedded_hal_impl {
                         };
                         SpidevTransfer::read_write(tx, buf)
                     }
+                    SpiOperation::DelayUs(us) => {
+                        SpidevTransfer::delay((*us).try_into().unwrap_or(u16::MAX))
```

Comment:
I chose to max out the delay to `u16::MAX` but that may not be what we want here. 
Let me know what you think.

Reply:
I think we can leave at that here but please document in this method and in the `Spidev` struct that delays are capped to 65535 microseconds.